### PR TITLE
Moved: Google.GoogleDrive version 79.0.2.0

### PR DESCRIPTION
**Pull request by SpecterShell**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] This PR only modifies one (1) manifest
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/116504&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/g/Google/GoogleDrive/79.0.2.0/Google.GoogleDrive.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Created with YamlCreate.ps1 v2.2.9 $debug=QUSU.CRLF.7-3-6.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Google.GoogleDrive
```
Comment (stephengillie) [inline manifests/g/Google/GoogleDrive/79.0.2.0/Google.GoogleDrive.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Google.Drive
```
Please update file path to match this change.

File: manifests/g/Google/GoogleDrive/79.0.2.0/Google.GoogleDrive.locale.zh-CN.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# Created with YamlCreate.ps1 v2.2.9 $debug=QUSU.CRLF.7-3-6.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.4.0.schema.json
+
+PackageIdentifier: Google.GoogleDrive
```
Comment (stephengillie) [inline manifests/g/Google/GoogleDrive/79.0.2.0/Google.GoogleDrive.locale.zh-CN.yaml:4]:
```suggestion
PackageIdentifier: Google.Drive
```
Please update file path to match this change.

File: manifests/g/Google/GoogleDrive/79.0.2.0/Google.GoogleDrive.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.2.9 $debug=QUSU.CRLF.7-3-6.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Google.GoogleDrive
```
Comment (stephengillie) [inline manifests/g/Google/GoogleDrive/79.0.2.0/Google.GoogleDrive.yaml:4]:
```suggestion
PackageIdentifier: Google.Drive
```
Please update file path to match this change.

File: manifests/g/Google/GoogleDrive/79.0.2.0/Google.GoogleDrive.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# Created with YamlCreate.ps1 v2.2.9 $debug=QUSU.CRLF.7-3-6.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Google.GoogleDrive
```
[the remark this answers is not in the document]
Comment (SpecterShell) [reply manifests/g/Google/GoogleDrive/79.0.2.0/Google.GoogleDrive.installer.yaml:4]: The package `Google.Drive` has been removed though. 🤔

[the remark this answers is not in the document]
Comment (stephengillie) [reply manifests/g/Google/GoogleDrive/79.0.2.0/Google.GoogleDrive.installer.yaml:4]: > The package `Google.Drive` has been removed though. 🤔

Oh, I was unaware. Sorry.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**